localization/pose_initializer/src/pose_initializer/pose_initializer_core.cpp

```diff
@@ -34,12 +35,14 @@ PoseInitializer::PoseInitializer() : Node("pose_initializer")
   output_pose_covariance_ = get_covariance_parameter(this, "output_pose_covariance");
   gnss_particle_covariance_ = get_covariance_parameter(this, "gnss_particle_covariance");
 
+  ekf_localization_trigger_ = std::make_unique<EkfLocalizationTriggerModule>(this);
```

Comment:
In the simple planning simulation, EKF is not launched, so it cannot be initialized. It needs to add the `ekf_enabled` to parameter.

Reply:
Sorry, I did not check the planning_simulator.

Rather than introducing the `ekf_enable` parameter, how about not triggering `ekf_localization_trigger_` when `(!ndt_enabled && !gnss_enabled)` is `true`?

Reply:
I don't know when EKF localizer is used. @YamatoAndo Is this condition appropriate?

Reply:
ok, I'll fix it.